src/Mvc/Mvc.Core/src/FromBodyAttribute.cs

```diff
@@ -23,6 +24,7 @@ public class FromBodyAttribute : Attribute, IBindingSourceMetadata, IConfigureEm
         /// The default behavior is to use framework defaults as configured by <see cref="MvcOptions.AllowEmptyInputInBodyModelBinding"/>.
         /// Specifying <see cref="EmptyBodyBehavior.Allow"/> or <see cref="EmptyBodyBehavior.Disallow" /> will override the framework defaults.
         /// </remarks>
+        // REVIEW: What should we do about this? Type forward EmptyBodyBehavior? Write analyzers to warn against configuring this with MapAction?
```

Comment:
I'm interested on feedback about this. Personally, I think this is simple enough to just implement.

Reply:
> Type forward EmptyBodyBehavior

That seems reasonable.